lang/cs/Org.Apache.REEF.Network.Tests/NamingService/NameServerTests.cs

```diff
@@ -233,6 +234,33 @@ public void TestConstructorInjection()
             }
         }
 
+        [TestMethod]
+        public void TestNameCache()
+        {
+            double interval = 1000;
+            var config =
+                TangFactory.GetTang()
+                    .NewConfigurationBuilder()
+                    .BindNamedParameter<NameCacheConfiguration.CacheEntryExpiryTime, double>(
+                        GenericType<NameCacheConfiguration.CacheEntryExpiryTime>.Class,
+                        interval.ToString(CultureInfo.InvariantCulture))
+                    .Build();
+            
+            var injector = TangFactory.GetTang().NewInjector(config);
+            var cache = injector.GetInstance<NameCache>();
+
+            cache.Set("dst1", new IPEndPoint(IPAddress.Any, 0));
+            Thread.Sleep(2000);
```

Comment:
Can we use a much shorter time-out? The test adds 2s to the test execution time right now, which can add up quickly, especially on build servers with fixed timeouts on them.